docs/guides/features/system-tray.md

```diff
@@ -193,5 +193,18 @@ Note that you need to add `icon-ico` or `icon-png` feature flag to the tauri dep
 app.tray_handle().set_icon(tauri::Icon::Raw(include_bytes!("../path/to/myicon.ico").to_vec())).unwrap();
 ```
 
+### Keep app running in the background after closing all windows
+
+By default, tauri closes the application when the last window is closed.
+If your app should run in the backgorund, you can call `api.prevent_close()` like so:
+
+```rust
+tauri::Builder::default()
+    .on_window_event(|event| if let tauri::WindowEvent::CloseRequested { api, .. } = event.event() {
+        event.window().hide().unwrap();
+        api.prevent_close();
+    })
```

Comment:
this hidesthe window or prevents closing the window where if you want to keep the app running without any windows, then a better example would be the example here https://docs.rs/tauri/latest/tauri/struct.App.html#method.run

Reply:
I agree, we can make it more minimal. I've updated the PR

Reply:
It is not about how minimal the example. The example you are using prevents the **window** from being closed, what you want instead is to prevent the **app** from being closed which is what the example I linked above shows.

Reply:
Whoops, you're very right, sorry. Should be good now.